dhall-openapi/dhall-openapi.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version:  >=1.10
 name:           dhall-openapi
-version:        0.1.0.0
+version:        0.1.2
```

Comment:
What's the reason for this bump?

Reply:
@sjakobi I first had 0.1.1 (to be sure I was above the previous fourth digit schema), then 0.1.2 on my second change (remove Turtle). Should I make it  `0.1.1` instead ?

Reply:
Also it would be nice to have someone replacing the current "maintainer" in the cabal file. Maybe @TristanCacqueray could jump in ;-)

Reply:
Thanks for the clarification! In general, Gabriel takes care of bumping the version numbers during the release process.

Since `dhall-openapi` hasn't been properly released at all so far, no bump seems necessary for now.

Reply:
Yeah, there's no need to bump the version until we release.  Also, the version number here is legacy from the `dhall-kubernetes` project.  When we do release the project it will be at version 1.0.0.  I typically don't do `0.*` releases of packages.

Reply:
I am not familiar with the maintainer duties and I think it is better if @Gabriel439 is the selected one for `dhall-openapi`.